### PR TITLE
1.3.0: version bump

### PR DESCRIPTION
**Pull request by cognifloyd**
Prepare for v1.3.0

Is there anything other than the version in galaxy.yml that we need to update?

### CODE REVIEW

**Comment by rhysmeister** [comment]
We'll also need to update changelogs/changelog.yaml then it's just a case of adding a tag against the master branch.

File: changelogs/changelog.yaml [hunk 1]
```diff
@@ -69,3 +69,28 @@ releases:
...
+      - 338 - role monogdb_repository - Variablize repository details.
+      - 345 - roles mongodb_config, mongodb_mongod, mongodb_mongos - Make security.keyFile configurable.
+      - 346 - roles mongodb_config, mongodb_mongod, mongodb_mongos - Allow using net.bindIpAll instead of net.bindIp.
+      - 347 - roles mongodb_config, mongodb_mongod, mongodb_mongos - Allow overriding net.compression.compressors in mongo*.conf
+    bugfixes:
+      - 312 - Set no_log True for ssl_keyfile.
+      - 315 - Fix exception handling for mongodb_stepdown module on python3.6
+      - 320 - Fix exception handling for modules mongodb_balancer, mongodb_shard, and mongodb_status.
+      - 352 - Add ansible.posix collection to dependencies list.
+    release_date: '2021-02-09'
```
Comment (rhysmeister) [inline changelogs/changelog.yaml:96]: release_date needs updating, the format is YYYY-MM-DD

**Comment by cognifloyd** [comment]
So, how do I regenerate CHANGELOG.rst from changelogs/changelog.yaml?

**Comment by rhysmeister** [comment]
> So, how do I regenerate CHANGELOG.rst from changelogs/changelog.yaml?

https://docs.ansible.com/ansible/devel/dev_guide/developing_collections_changelogs.html

Looks like I forget the last couple of times. I recall having a lot of problems using the ansibul tool and I think I might have resorted to copy/paste.

**Comment by cognifloyd** [comment]
K. I think this is ready to merge once tests pass. Then we can tag the release.

**Pull request merged**